src/node_jsvmapi.h

```diff
@@ -151,6 +157,8 @@ NODE_EXTERN napi_value napi_create_type_error(napi_env e, napi_value msg);
 
 // Methods to get the the native napi_value from Primitive type
 NODE_EXTERN napi_valuetype napi_get_type_of_value(napi_env e, napi_value v);
+NODE_EXTERN bool napi_is_int32(napi_env e, napi_value v);
+NODE_EXTERN bool napi_is_uint32(napi_env e, napi_value v);
```

Comment:
While V8 treats certain numbers as int32 or uint32, that is a V8-specific optimization that I'm not sure we should expose in NAPI.